src/styles/main.scss

```diff
@@ -0,0 +1,81 @@
+$day-bg-color: #eee;
+$day-size: 100px;
+$gap-size: 1px;
+$font-size: 30px;
+$font-color: #000;
+$hover-color: #ffbfcb;
+$widthCalendar: 7 * $day-size + $gap-size * 6;
+
+.body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  height: 100vh;
+  align-items: center;
+}
+
+.calendar {
+  width: $widthCalendar;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size;
+
+  $days: (
+    mon: 0,
+    tue: 1,
+    wed: 2,
+    thu: 3,
+    fri: 4,
+    sat: 5,
+    sun: 6,
+  );
+
+  @each $day, $i in $days {
+    &--start-day-#{$day} {
+      .calendar__day:nth-child(1) {
+        margin-left: ($day-size * ($i) + ($day-size * ($i) / 100));
```

Comment:
The calculation for `margin-left` seems unnecessarily complex. You can simplify it by using `margin-left: $day-size * $i;` since the additional calculation doesn't seem to add any meaningful difference.